x-pack/plugins/secops/public/components/horizontal_bar_chart/index.tsx

```diff
@@ -33,16 +33,30 @@ interface HorizontalBarChartProps {
 export const HorizontalBarChart = pure<HorizontalBarChartProps>(
   ({ barChartdata, width, height, title, loading }) => {
     return loading ? (
-      <LoadingPanel height="100%" width="100%" text="Loading data" />
+      <LoadingPanel height="auto" width="100%" text="Loading data" />
     ) : (
-      <>
+      <Container>
         <EuiTitle size="s">
           <h3>{title}</h3>
         </EuiTitle>
         <EuiSeriesChart width={width} height={height} yType="ordinal" orientation="horizontal">
           <EuiBarSeries name="Tag counts" data={barChartdata} />
         </EuiSeriesChart>
-      </>
+      </Container>
     );
   }
 );
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  height: auto;
+  & > div {
+    .rv-xy-plot {
+      svg {
+        height: fit-content !important;
```

Comment:
I would add a comment about why we have a `!important` for  the next poor soul who wonders in here if something breaks or gets weird.

I don't know if this is helpful but sometimes I use a styled `&& {` and that gets me further.

Reply:
that's the problem from eui
![image](https://user-images.githubusercontent.com/189600/49651067-33be8980-f9fc-11e8-9682-ffadd8260372.png)